hash/saturn.xml

```diff
@@ -34347,7 +34347,7 @@ Sega logo animation draws offset [VDP2]
 
 	<!-- Identifying Saturn Power N.1 EUR [610-6576] - Core Demo Disc EUR [790-0002-50]... -->
 	<software name="satpowr1" supported="no">
-		<description>Saturn Power No. 1 (Euro) ~ Core Demo Disc (Euro)</description>
+		<description>Saturn Power No. 1 (Europe) ~ Core Demo Disc (Europe)</description>
```

Comment:
This has a duplicated dual tag.

Reply:
It’s because it’s two titles used in Europe both jammed into the description.  One of them should be moved to an `alt_title`.